### ISSUE TITLE
get_api_group() doesn't work

### ISSUE THREAD

**Issue by lukas-bednar** [kubevirt/client-python#4]
When running following snippet of code
```python
from kubernetes import config
import kubevirt

def get_client(kubeconfig="kubeconfig"):
    cl = config.kube_config._get_kube_config_loader_for_yaml_file(kubeconfig)
    cl.load_and_set(kubevirt.configuration)
    return kubevirt.DefaultApi()

a = get_client()
a..get_api_group()
```
I am getting following error
```
In [8]: a.get_api_group()
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-8-c822db971437> in <module>()
----> 1 a.get_api_group()

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/apis/default_api.pyc in get_api_group(self, **kwargs)
   1279             return self.get_api_group_with_http_info(**kwargs)
   1280         else:
-> 1281             (data) = self.get_api_group_with_http_info(**kwargs)
   1282             return data
   1283 

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/apis/default_api.pyc in get_api_group_with_http_info(self, **kwargs)
   1349                                         _preload_content=params.get('_preload_content', True),
   1350                                         _request_timeout=params.get('_request_timeout'),
-> 1351                                         collection_formats=collection_formats)
   1352 
   1353     def get_api_resources(self, **kwargs):

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/api_client.pyc in call_api(self, resource_path, method, path_params, query_params, header_params, body, post_params, files, response_type, auth_settings, callback, _return_http_data_only, collection_formats, _preload_content, _request_timeout)
    324                                    body, post_params, files,
    325                                    response_type, auth_settings, callback,
--> 326                                    _return_http_data_only, collection_formats, _preload_content, _request_timeout)
    327         else:
    328             thread = threading.Thread(target=self.__call_api,

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/api_client.pyc in __call_api(self, resource_path, method, path_params, query_params, header_params, body, post_params, files, response_type, auth_settings, callback, _return_http_data_only, collection_formats, _preload_content, _request_timeout)
    159             # deserialize response data
    160             if response_type:
--> 161                 return_data = self.deserialize(response_data, response_type)
    162             else:
    163                 return_data = None

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/api_client.pyc in deserialize(self, response, response_type)
    237             data = response.data
    238 
--> 239         return self.__deserialize(data, response_type)
    240 
    241     def __deserialize(self, data, klass):

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/api_client.pyc in __deserialize(self, data, klass)
    277             return self.__deserialize_datatime(data)
    278         else:
--> 279             return self.__deserialize_model(data, klass)
    280 
    281     def call_api(self, resource_path, method,

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/api_client.pyc in __deserialize_model(self, data, klass)
    629                 kwargs[attr] = self.__deserialize(value, attr_type)
    630 
--> 631         instance = klass(**kwargs)
    632 
    633         return instance

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/models/v1_api_group.pyc in __init__(self, api_version, kind, name, preferred_version, server_address_by_client_cid_rs, versions)
     68         if preferred_version is not None:
     69           self.preferred_version = preferred_version
---> 70         self.server_address_by_client_cid_rs = server_address_by_client_cid_rs
     71         self.versions = versions
     72 

/home/lbednar/work/kubevirt-org/kubevirt-pysdk-sandbox/E/lib/python2.7/site-packages/kubevirt/models/v1_api_group.pyc in server_address_by_client_cid_rs(self, server_address_by_client_cid_rs)
    186         """
    187         if server_address_by_client_cid_rs is None:
--> 188             raise ValueError("Invalid value for `server_address_by_client_cid_rs`, must not be `None`")
    189 
    190         self._server_address_by_client_cid_rs = server_address_by_client_cid_rs

ValueError: Invalid value for `server_address_by_client_cid_rs`, must not be `None`
```

**Comment by lukas-bednar** [comment]
This issue was addressed here kubevirt/kubevirt#547 , there is no real fix which can be done on kubevirt side, at least there is workaround for that.

```python
from kubevirt.models.v1_api_group import V1APIGroup

from kubernetes import config
import kubevirt

import pprint

def get_client(kubeconfig="kubeconfig"):
    cl = config.kube_config._get_kube_config_loader_for_yaml_file(kubeconfig)
    cl.load_and_set(kubevirt.configuration)

    api = kubevirt.DefaultApi()

    return api

def fix_api_group_null_value():
    _server_address_by_client_cid_rs = V1APIGroup.server_address_by_client_cid_rs

    @_server_address_by_client_cid_rs.setter
    def _server_address_by_client_cid_rs(self, server_address_by_client_cid_rs):
        self._server_address_by_client_cid_rs = server_address_by_client_cid_rs

    V1APIGroup.server_address_by_client_cid_rs = _server_address_by_client_cid_rs

if __name__ == "__main__":
    fix_api_group_null_value()
    c = get_client()
    pprint.pprint(c.get_api_group())
```

**Comment by pkliczewski** [comment]
not needed anymore
